src/main/java/Entities/Schedule.java

```diff
@@ -5,9 +5,10 @@
 public class Schedule {
 
     // todo: make a toString method for Entities.Event to make this work better
-    private HashMap<String, HashMap<String, String>> schedule;  // day, start hour, event name
+    //NOTE: had to make schedule public to access it in ScheduleEditor
+    public HashMap<String, HashMap<String, String>> schedule;  // day, start hour, event name
```

Comment:
instance attributes definitely should not be public

Reply:
I made ScheduleEditor use the getter method from Schedule now